src/main/resources/tags/links/rpconvert.tag

```diff
@@ -4,3 +4,10 @@ type: text
 
 Use this tool to convert your Java resource pack into a Bedrock resource pack to use in Geyser's `packs` folder.
 https://rtm516.github.io/ConvertJavaTextureToBedrock/
+Unfortunately, this tool is not updated to support packs for versions newer than 1.17.
+
+To convert a resource pack with CustomModelData, use the following converter: https://github.com/Kas-tle/java2bedrock.sh 
+For more info on how to use it, see the pinned messages in "Custom Resource Pack Help". Be advised: Converting a pack to make it bedrock compatible, espescially with CustomModelData isn't always an automated process. If you need help, ask in the aforementioned channel!
```

Comment:
```suggestion
For more info on how to use it, see the pinned messages in "Custom Resource Pack Help". Be advised: Converting a pack to make it bedrock compatible, especially with CustomModelData isn't always an automated process. If you need help, ask in the aforementioned channel!
```
Also can we make "Custom Resource Pack Help" a channel link? Its a thread so people will probably have a hard time finding it